src/memory_targets/memory_target_numa.c

```diff
@@ -189,27 +189,31 @@ static umf_result_t query_attribute_value(void *srcMemoryTarget,
                                           memattr_type_t type) {
     hwloc_topology_t topology = umfGetTopology();
     if (!topology) {
+        LOG_ERR("umfGetTopology() failed");
```

Comment:
nit: Instead of `function() failed` pattern, more often we use more descriptive log messages like `Retrieving cached topology failed`.

Reply:
also you should use LOG_P for funcion which sets errno

Reply:
Done